Mobile Buy SDK/Mobile Buy SDK/Models/BUYCheckout.h

```diff
@@ -225,7 +220,7 @@
 @property (nonatomic, copy, readonly) NSString *sourceName;
 
 /**
- *  The unique identifier for the source: the channelId
+ *  The unique identifier for the source: the appId
  */
 @property (nonatomic, copy, readonly) NSString *sourceIdentifier;
```

Comment:
Ok, leave it to mobile_app for now, but I believe this at one point will no longer be necessary. No ETA on this though.